lib/dpl/provider/bintray.rb

```diff
@@ -0,0 +1,515 @@
+require 'json'
+require 'net/http'
+require "uri"
+require 'find'
+
+module DPL
+  class Provider
+    class Bintray < Provider
+
+      @file
+      @user
+      @key
+      @passphrase
+      @url
+      @descriptor
+      @dry_run
+      @test_mode = false
+
+      def check_auth
+      end
+
+      def needs_key?
+        false
+      end
+
+      def set_test_mode
```

Comment:
Did I mention elsewhere that `set_*` and `get_*` method names are not very Ruby-like?

The method should be `test_mode=(bool)`, and not necessarily assume it's going to set to `true` and never to `false` (or whatever it might be).

For example:

```ruby
def test_mode=(bool)
  @test_mode = bool
end
```

In general, though, if the instance variable name and the method names are "similar", you can do away with:

```ruby
attr_writer :test_mode
```